code/modules/mob/living/carbon/life.dm

```diff
@@ -821,3 +822,12 @@
 		return
 
 	heart.beating = !status
+
+//////////////////////
```

Comment:
don't do this. it ruins auto doc and looks horrible

Reply:
Actually document what this does and when it is called

Reply:
Done, I also moved it to the area with all the other handler procs.